src/software/sensor_fusion/sensor_fusion.h

```diff
@@ -19,35 +19,31 @@
 
 /**
  * Sensor Fusion is an abstraction around all filtering operations that our system may
- * need to perform. It produces Worlds that may be used, and consumes SensorMsgs
- *
- * This produce/consume pattern is performed by extending both "Observer" and
- * "Subject". Please see the implementation of those classes for details.
+ * need to perform. It produces Worlds that may be used, and consumes vision detections,
+ * refbox data, and robot statuses
  */
-class SensorFusion : public Subject<World>, public ThreadedObserver<SensorMsg>
+class SensorFusion
```

Comment:
I think that there are includes in this file that can be cleaned up

Reply:
yup, removed unneeded observer / threading headers